cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -718,4 +719,157 @@ describe('Dashboards', () => {
     cy.getByTestID('button-copy').click()
     cy.getByTestID('notification-success--children').should('be.visible')
   })
+  it('changes time range', () => {
+    const dashName = 'dashboard'
+    const newDate = new Date()
+    const now = newDate.toISOString()
+    const pastDate = new Date(new Date() - 1000 * 60 * 60 * 2)
+    const past = pastDate.toISOString()
+    cy.get<Organization>('@org').then(({id}: Organization) =>
+      cy.createDashboard(id, dashName).then(({body}) => {
+        cy.createCell(body.id).then(cell1Resp =>
+          cy.createView(body.id, cell1Resp.body.id).then(() =>
+            cy.fixture('routes').then(({orgs}) =>
+              cy.get<Organization>('@org').then(({id}: Organization) => {
+                cy.visit(`${orgs}/${id}/dashboards-list`)
+                cy.getByTestID('tree-nav').then(() =>
+                  cy
+                    .intercept('POST', '/api/v2/query?*')
+                    .as('loadQuery')
+                    .then(() =>
+                      cy.writeLPData({
+                        lines: genCurve({points: 3500, freq: 6, shift: 13}),
+                        offset: `${8 * 24 * 60}m`,
+                        stagger: `${(8 * 24 * 60) / 3499}m`,
+                      })
+                    )
+                )
+              })
+            )
+          )
+        )
+      })
+    )
+    cy.getByTestID('dashboard-card--name').click()
+    cy.getByTestID('cell-context--toggle').click()
+    cy.getByTestID('cell-context--configure').click()
+    cy.getByTestID('selector-list curve').click()
+    cy.getByTestID('selector-list p').click()
+    cy.getByTestID('time-machine-submit-button').click()
+    cy.getByTestID('save-cell--button').click()
+    cy.wait('@loadQuery')
+
+    const snapshot = makeGraphSnapshot()
+
+    // fixed time range values
+    // past 1m
+    cy.getByTestID('timerange-dropdown').click()
+    cy.getByTestID('dropdown-item-past1m').click()
+    cy.wait('@loadQuery')
+
+    const snapshot2 = makeGraphSnapshot()
+    snapshot2.shouldBeSameAs(snapshot, false)
+
+    // past 5m
+    cy.getByTestID('timerange-dropdown').click()
+    cy.getByTestID('dropdown-item-past5m').click()
+    cy.wait('@loadQuery')
+
+    const snapshot3 = makeGraphSnapshot()
+    snapshot3.shouldBeSameAs(snapshot2, false)
+
+    /*
+    // past 15m
+    cy.getByTestID('timerange-dropdown').click()
```

Comment:
Remove commented code please

Reply:
I pushed another commit where the code has been added to the test. This particular feature caused the test to fail on CircleCI. After moving the 15m time range test to the bottom, the issue didn't appear anymore.

Reply:
Commented code was a part which made the test fail. I moved it to the end of the test. It doesn't cause any problems anymore.